soc/riscv/riscv-ite/common/chip_chipregs.h

```diff
@@ -1678,8 +1678,23 @@ struct wdt_it8xxx2_regs {
 
 #define IT8XXX2_GPIO_GCR        ECREG(IT8XXX2_GPIO_BASE + 0x00)
 
+#define IT8XXX2_GPIO_GCR25      ECREG(IT8XXX2_GPIO_BASE + 0xD1)
+#define IT8XXX2_GPIO_GCR26      ECREG(IT8XXX2_GPIO_BASE + 0xD2)
+#define IT8XXX2_GPIO_GCR27      ECREG(IT8XXX2_GPIO_BASE + 0xD3)
+#define IT8XXX2_GPIO_GCR28      ECREG(IT8XXX2_GPIO_BASE + 0xD4)
+#define IT8XXX2_GPIO_GCR31      ECREG(IT8XXX2_GPIO_BASE + 0xD5)
+#define IT8XXX2_GPIO_GCR32      ECREG(IT8XXX2_GPIO_BASE + 0xD6)
+#define IT8XXX2_GPIO_GCR33      ECREG(IT8XXX2_GPIO_BASE + 0xD7)
+#define IT8XXX2_GPIO_GCR19      ECREG(IT8XXX2_GPIO_BASE + 0xE4)
+#define IT8XXX2_GPIO_GCR20      ECREG(IT8XXX2_GPIO_BASE + 0xE5)
+#define IT8XXX2_GPIO_GCR21      ECREG(IT8XXX2_GPIO_BASE + 0xE6)
+#define IT8XXX2_GPIO_GCR22      ECREG(IT8XXX2_GPIO_BASE + 0xE7)
+#define IT8XXX2_GPIO_GCR23      ECREG(IT8XXX2_GPIO_BASE + 0xE8)
+#define IT8XXX2_GPIO_GCR24      ECREG(IT8XXX2_GPIO_BASE + 0xE9)
+#define IT8XXX2_GPIO_GCR30      ECREG(IT8XXX2_GPIO_BASE + 0xED)
+#define IT8XXX2_GPIO_GCR29      ECREG(IT8XXX2_GPIO_BASE + 0xEE)
+/* TODO: correct GRC1 to GCR1 */
 #define IT8XXX2_GPIO_GRC1       ECREG(IT8XXX2_GPIO_BASE + 0xF0)
-#define IT8XXX2_GPIO_GRC21      ECREG(IT8XXX2_GPIO_BASE + 0xE6)
```

Comment:
Need to change the code that referred to this.

> /workdir/zephyr/soc/riscv/riscv-ite/it8xxx2/soc.c:36:2: error: 'IT8XXX2_GPIO_GRC21' undeclared (first use in this function); did you mean 'IT8XXX2_GPIO_GRC1'?
>    36 |  IT8XXX2_GPIO_GRC21 &= ~(BIT(0) | BIT(1));
>       |  ^~~~~~~~~~~~~~~~~~
>       |  IT8XXX2_GPIO_GRC1

Reply:
Leaving my approved because this has to be fixed in order to get the required checks to complete.